### DISCUSSION TITLE
Question 50/53: how to archive videos in EMu [ZK - Action]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#602] [Q&A] [answered]
Description: Is there a workflow for this?

**Comment by rondlg** [comment]
Response: See github issues:

- https://github.com/fieldmuseum/EMu-Documentation/discussions/322
- https://github.com/fieldmuseum/EMu-Documentation/discussions/323- 

Follow-up: Ask @magpiedin about proper documentation (camera release) for archiving video.

**Comment by magpiedin** [comment, marked as answer]
Yes, same rules as for still photography. 
Format should be "mp4" with .h264 encoding (let us know if you need help checking/converting). 

Keep an eye on filesize limits please. Let us know if they are HUGE before they go in.
- Stage files on Imaging_Scratch with a [footage log event record](https://github.com/fieldmuseum/EMu-Documentation/discussions/321).
- Filesize on the main ‘Multimedia’ tab should be < 200MB(ish)
- Larger HD video can go on the Supplementary tab. 
supplementary tab. 

Multimedia Workflows are here: https://github.com/fieldmuseum/EMu-Documentation/wiki/emultimedia-workflows
